.github/workflows/docker.yaml

```diff
@@ -89,6 +84,79 @@ jobs:
             type=raw,value=${{ github.event.pull_request.head.ref }}
             type=raw,value=stable,enable=${{ startsWith(github.event.ref, 'refs/tags/v') }}
 
+      - name: Build and push livepeer docker image
+        uses: docker/build-push-action@v4
+        with:
+          build-args: |
+            BUILD_TAGS=${{ steps.build-tag.outputs.build-tags }}
+          # linux/arm64 is disabled until
+          # <https://github.com/livepeer/go-livepeer/issues/2545> gets
+          # resolved
+          context: .
+          platforms: linux/amd64, linux/arm64
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}
+          file: "docker/Dockerfile"
+          labels: ${{ steps.meta.outputs.labels }}
+          cache-from: type=registry,ref=livepeerci/build:cache
+          cache-to: type=registry,ref=livepeerci/build:cache,mode=max
+
+  builder:
+    name: go-livepeer builder docker image generation
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository
+    permissions:
+      packages: write
+      contents: read
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3.5.3
+        with:
+          fetch-depth: 0
+          # Check https://github.com/livepeer/go-livepeer/pull/1891
+          # for ref value discussion
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Cleanup hosted runner
+        run: |
+          sudo apt purge -yqq dotnet-* mono-* llvm-* adoptopenjdk-* libllvm* powershell* openjdk-* \
+          temurin-* mongodb-* firefox mysql-* \
+          hhvm google-chrome-stable \
+          libgl1-mesa-dri microsoft-edge-stable \
+          google-cloud-sdk azure-cli
+          sudo apt autoremove -y
+          sudo rm -rf /usr/share/dotnet /usr/local/lib/android
```

Comment:
Do we need to remove it both here and in line 40?

Reply:
not really! but just keeping it here for future-proofing against having similar issues (out of disk space on github runner). go-livepeer won't be needing android libraries (and other packages being removed).